internal/k8shandler/collection.go

```diff
@@ -93,6 +95,18 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 		return err
 	}
 
+	var httpInputs []string
```

Comment:
```
httpInputs := sets.NewString()
...
httpInputs.Insert(input.Name)
```
? Make the collection of HTTP inputs a set from the start and avoid copying in RemoveInputServices()?

Reply:
Done